tests/expressions/tests.py

```diff
@@ -2849,3 +2849,39 @@ def test_nulls_false(self):
             F("field").asc(nulls_first=False)
         with self.assertRaisesMessage(ValueError, msg):
             F("field").desc(nulls_last=False)
+
+
+class QueryExpressionTestCase(TestCase):
+    def setUp(self):
```

Comment:
It's more common to see setUpTestData in Django's test suite for performance reasons. You only have one test method here now, but it could be added to over time?